extensions/extensions-helm/aissemble-configuration-store-chart/templates/webhook-cleanup-job.yaml

```diff
@@ -0,0 +1,46 @@
+{{ if .Values.webhook.enable }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ index .Values "aissemble-quarkus-chart" "app" "name" }}-webhook-cleanup
+  annotations:
+    "helm.sh/hook": pre-delete, pre-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: {{ index .Values "aissemble-quarkus-chart" "app" "name" }}-webhook-cleanup
+    spec:
+      serviceAccountName: {{ .Values.webhook.serviceAccount.name }}
+      restartPolicy: OnFailure
+      containers:
+      - name: clean-webhook
+        image: {{ index .Values "aissemble-quarkus-chart" "deployment" "image" "repo" }}{{ index .Values "aissemble-quarkus-chart" "deployment" "image" "name" }}:{{ index .Values "aissemble-quarkus-chart" "deployment" "image" "tag" | default .Chart.AppVersion }}
```

Comment:
A: This file might look a bit tidier by using `with` and/or local variables.  Like these image lines becomes a bit more readable if you wrap it in `{{- with .Values "aissemble-quarkus" "deployment" "image" }}`

Reply:
Refactored cleanup and init to use helm variables